app/Repositories/Eloquents/AbstractEloquentRepository.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Repositories\Eloquents;
+
+abstract class AbstractEloquentRepository
+{
+	abstract public function model();
+}
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/AbstractEloquentRepository.php</br>Spaces must be used to indent lines; tabs are not allowed